runner/migrations/0013_merge_20200117_2050.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.8 on 2020-01-17 20:50
```

Comment:
We have this on develop branch now https://github.com/mskcc/beagle/blob/develop/runner/migrations/0013_merge_20200117_1842.py. I am not sure can duplicate cause us problems now?